index.js

```diff
@@ -62,7 +120,192 @@ function createTaskCreatedAt(taskCreatedAt) {
 function createTaskCheckBox() {
   return `<input
             type="checkbox"
-            class="table-box-check-box"
             name="checkbox-done"
           /> `;
 }
+
+function sortTasks(typeOfSort) {
+  if (typeOfSort == "a-to-z") {
+    sortTasksAtoZ();
```

Comment:
Would call this `sortTasksByTitle` instead.

Reply:
got it